latex-guide/latex-guide.md

```diff
@@ -0,0 +1,814 @@
+---
+title: '10 Things to Know About Writing Academic Papers in LaTeX'
+author: Jake Bowers and Luke Olson
+link-citations: true
+numbersections: true
+colorlinks: true
+secnumdepth: 2
+number-sections: true
+bibliography: latex-guide.bib
+linkReferences: true
+listings: true
+html_document:
+  toc: true
+  toc_depth: 1
+  number-sections: true
+  listings: true
+  theme: journal
+  highlight: pygment
+  pandoc_args: ["--from= markdown+autolink_bare_uris+tex_math_single_backslash+inline_code_attributes"]
+---
+
+Writing is a time-consuming process; writing high-quality publications requires
+attention to detail at every step of the way, from the actual prose on paper to
+its layout in the document to the presentation of figures.  In this guide we
+walk you through 10 aspects of writing a scientific article using
+[LaTeX](https://en.wikipedia.org/wiki/LaTeX) to format your work and save you
+time.  We emphasize typing commands at the [unix command
+line](https://en.wikipedia.org/wiki/Unix_shell) in this guide as a way for you
+to peek under the hood of the LaTeX engine. This will give you (the author!)
+power over the production of your own academic documents.^[We have decided to
+write this guide in a very opinionated way. And we emphasize the nitty gritty of
```

Comment:
I know that this guide is meant to be for a fairly technical audience. In case people come to it looking for more of an introductory LaTeX how-to, suggest making it clearer in the intro that this is not an intro-level guide, and perhaps linking to 1 or 2 of those that you recommend.